dist/style.css

```diff
@@ -59,3 +55,288 @@ body {
   -ms-transform: translate(-50%, -50%);
   transform: translate(-50%, -50%);
 }
+
+/* container css for most high level containers  */
+.container-parent {
```

Comment:
When I view this page in full-screen mode, everything in `.container-parent` feels very wide. Maybe we should try having a `max-width`, or a static amount of `margin-left` and `margin-right`, to prevent it from over-stretching?

Reply:
changed `.parent-container` to have a `max-width` of **1000px**